lua/nvim-treesitter/parsers.lua

```diff
@@ -973,6 +973,14 @@ list.llvm = {
   maintainers = { "@benwilliamgraham" },
 }
 
+list.liquidsoap = {
+  install_info = {
+    url = "https://github.com/savonet/tree-sitter-liquidsoap",
+    files = { "src/parser.c", "src/scanner.cc" },
```

Comment:
not a blocker for this PR, but it'd be nice if you could rewrite the scanner in C

Reply:
I'm using the C++ regexp support in it. Waiting for this to switch to C: https://github.com/tree-sitter/tree-sitter/issues/2620